### PR TITLE
Push umbrella roadmap tweaks

### PR DESCRIPTION
**Pull request by matzew**
some more cleaning up, and adding future SPS and WPS releases to the 'unscheduled' section of our roadmap

### CODE REVIEW

**Comment by matzew** [comment]
@danbev @sebastienblanc please review and merge

File: docs/planning/index.markdown [hunk 1]
```diff
@@ -43,6 +43,22 @@ Dealing with conflicts while dealing with data and their state.
...
+### AeroGear SimplePush Server
+
+Next releases for our implementation of Mozilla's SimplePush specification:
+
+* [0.13.0](https://issues.jboss.org/browse/AGPUSH/fixforversion/12326562/?selectedTab=com.atlassian.jira.jira-projects-plugin:version-issues-panel)
+* [future tasks](https://issues.jboss.org/browse/AGPUSH/fixforversion/12326563/?selectedTab=com.atlassian.jira.jira-projects-plugin:version-issues-panel)
+
+### AeroGear WebPush Server
+
+The [WebPush Specificiation](https://tools.ietf.org/html/draft-thomson-webpush-http2) is in an early state, but we see it as the successor of SimplePush. Below a few milestones for our WebPush effort are listed:
```
Comment (sebastienblanc) [inline docs/planning/index.markdown]: typo on specification

**Comment by matzew** [comment]
done and rebased!

**Comment by matzew** [comment]
@danbev @sebastienblanc fixed typo and rebased!

**Comment by sebastienblanc** [comment]
:+1:

**Comment by danbev** [comment]
:+1: Merged as a76a47d7d8a3e5dbb9487cd927ca3798772d18b7

**Pull request merged**